src/test/java/uk/gov/pay/api/json/CreatePaymentRequestDeserializerTest.java

```diff
@@ -114,8 +115,8 @@ public void deserialize_shouldDeserializeARequestWithAnAgreementId() throws Exce
         assertThat(paymentRequest.getAmount(), is(27432));
         assertThat(paymentRequest.getReference(), is("Some reference"));
         assertThat(paymentRequest.getDescription(), is("Some description"));
-        assertThat(paymentRequest.getReturnUrl(), is(nullValue()));
-        assertThat(paymentRequest.getAgreementId(), is("abc123"));
+        assertThat(paymentRequest.getReturnUrl(), is(Optional.empty()));
```

Comment:
Another option might be `assertThat(paymentRequest.getReturnUrl().isPresent(), is(false));`

Reply:
Yep, many ways to skin a cat! 🐈